test/Carthage/testdata/testproject/Cartfile.resolved

```diff
@@ -1,3 +1,4 @@
 github "Quick/Nimble" "v7.1.3"
 github "Swinject/Swinject" "2.4.1"
 github "erikdoe/ocmock" "v3.4.2"
+github "https://github.example.com/someOwner/someRepo" "0.0.1"
```

Comment:
You should add cases for `http://` and `git://` to show that we support these other types.

Reply:
fixed in https://github.com/fossas/fossa-cli/pull/989/commits/6f29ed4be011e96c4770c62922f3e74ae423f205